src/components/calcite-shell/usage/advanced.md

```diff
@@ -0,0 +1,115 @@
+          <!-- icon -->
+        </calcite-action>
+      </calcite-block-content>
+    </calcite-block>
+    <calcite-block collapsible  heading="Additional Block" summary="Baby shark doo doo doo doo.">
+      <calcite-block-content>
+          <p>Cool thing.</p>
+      </calcite-block-content>
+    </calcite-block>
+  </calcite-shell-panel>
+
+   <calcite-shell-panel slot="contextual-panel" layout="trailing" detached detached-scale="l">
+      <calcite-action-bar slot="action-bar">
+        <calcite-action-group>
+          <calcite-action text="Add" active>
+            <!-- icon -->
+          </calcite-action>
+          <calcite-action text="Save" disabled>
+            <!-- icon -->
+          </calcite-action>
+          <calcite-action text="Layers">
+            <!-- icon -->
+          </calcite-action>
+        </calcite-action-group>
+        <calcite-action-group>
+          <calcite-action text="Add">
+            <!-- icon -->
+          </calcite-action>
+          <calcite-action text="Save" disabled>
+            <!-- icon -->
+          </calcite-action>
+          <calcite-action text="Layers">
+            <!-- icon -->
+          </calcite-action>
+        </calcite-action-group>
+        <calcite-action-group slot="bottom-actions">
+          <calcite-action text="Tips" id="tip-manager-button">
+            <!-- icon -->
+          </calcite-action>
+        </calcite-action-group>
+      </calcite-action-bar>
+      <calcite-flow>
+        <calcite-flow-item heading="Layer settings">
+          <calcite-block collapsible open heading="Contextual Content" summary="Select goodness">
+            <calcite-value-list id="one" multiple filter-enabled>
+              <calcite-value-list-item text-label="2018 Population Density (Esri)" text-description="{POPDENS_CY}" value="POPDENS_CY">
+                <calcite-action slot="secondaryAction" id="action-test">
+                  <!-- icon -->
+                </calcite-action>
+              </calcite-value-list-item>
+              <calcite-value-list-item text-label="2018 Population Density [Updated]" text-description="{POPDENS_CY}" value="POPDENS_CY2">
+                <calcite-action slot="secondaryAction" id="action-test">
+                  <!-- icon -->
+                </calcite-action>
+              </calcite-value-list-item>
+              <calcite-value-list-item text-label="2018 Total Households (Esri)" text-description="{TOTHH_CY}" value="TOTHH_CY">
+                <calcite-action slot="secondaryAction" id="action-test">
+                </calcite-action>
+              </calcite-value-list-item>
+            </calcite-value-list>
+          </calcite-block>
+        </calcite-flow-item>
+      </calcite-flow>
+  </calcite-shell-panel>
+  <calcite-tip-manager slot="tip-manager" id="tip-manager">
+    <calcite-tip heading="The Red Rocks and Blue Water" text-thumbnail="This is an image of nature.">
```

Comment:
The `text-thumbnail` doesn't show up unless there is a `thumbnail` defined. Maybe we should add it for these?

Can you try copying these snippets to a HTML document and see if everything looks good and nothing is confusing for a user?

Reply:
sounds good. I just took out the thumbnails because they were place holders, ha! but just put them back in and checked the code and i think it looks good enough for a rough sample